test/integration/get-communities-controller.spec.ts

```diff
@@ -364,6 +364,168 @@ test('Get Communities Controller', function ({ components, spyComponents }) {
         })
       })
 
+      describe('when filtering by roles', () => {
```

Comment:
Would you mind checking the test cases that describe test scenarios in the `it` instead of the `describe`?